javaparser-core/src/main/java/com/github/javaparser/ast/modules/ModuleExportsStmt.java

```diff
@@ -161,4 +164,9 @@ public void ifModuleExportsStmt(Consumer<ModuleExportsStmt> action) {
     public Optional<ModuleExportsStmt> toModuleExportsStmt() {
         return Optional.of(this);
     }
+
+    public ModuleExportsStmt addModuleName(String name) {
```

Comment:
Was he naughty that he doesn't get some docs? :-)

Reply:
I can't think of a doc that would add anything, so I let it speak for itself :-)

Reply:
Ok, but while we are at self-explaining, I personally don't find the `moduleName` name pretty self-explanatory per-se. What is the module name(s) of an export statement? I know it now as I just tried it in absence of any other possibility to set the `to` part, but it was more trial & error than a self-explanatory name. ;-)
That mandates for a rename though, not for added JavaDoc. :-D

Reply:
Ok, just thought because `ModuleDeclaration#addDirective(String)'  got docs, but that is probably because it is a string that has to be properly formatted. :-)

Reply:
Yes, here the type gives it away.